src/views/onboarding/Onboarding.tsx

```diff
@@ -139,7 +139,7 @@ function OnboardingComponent({ productId }: { productId: string }) {
 
     const postLegalsResponse = await fetchWithLogs(
       { endpoint: 'ONBOARDING_POST_LEGALS', endpointParams: { institutionId, productId } },
-      { method: 'POST', data: { users, billingData, organizationType, origin } },
+      { method: 'POST', data: { billingData, institutionType, origin, users } },
```

Comment:
billingData model is not aligned to the model expected by the REST service
Check together with @esignore if the model of billing is the same of the model that she is creating